platform/saivpp/vpplib/SwitchStateBaseRoute.cpp

```diff
@@ -218,7 +218,7 @@ sai_status_t SwitchStateBase::addIpRoute(
 {
     SWSS_LOG_ENTER();
     bool isLoopback = false;
-    isLoopback = process_interface_loopback(serializedObjectId, true, isLoopback);
+    process_interface_loopback(serializedObjectId, isLoopback, true);
```

Comment:
The method params isLoopback and is_add got swapped, that cause compilation failure.